client/src/components/campaignDisplay/GearTab.tsx

```diff
@@ -18,8 +20,33 @@ const GearTab = ({ campaignData, dbRefetch }: GearTabProps) => {
         },
         { manual: true },
     );
+    console.log(campaignData.gear);
+
     return (
         <Flex direction="row" wrap="wrap" gap="3">
+            <TwilightSearchPopup
+                buttonText="Add Gear"
+                labelText="Search Gear"
+                options={Object.keys(gearData)}
+                onSubmit={(gearToAdd) => {
+                    console.log(gearToAdd);
+                    const newGearLocation = gearData[gearToAdd as keyof GearKeys].location;
+                    const updatedGear = { ...campaignData.gear };
+                    if (!campaignData.gear[newGearLocation]) {
+                        updatedGear[newGearLocation] = { [gearToAdd]: 0 };
+                    } else if (campaignData.gear[newGearLocation] && !campaignData.gear[newGearLocation][gearToAdd]) {
```

Comment:
&& !campaignData.gear[newGearLocation] is redundant here

Reply:
deleted campaignData.gear[newGearLocation] && instead